examples/foreach/foreach.bal

```diff
@@ -3,31 +3,34 @@ import ballerina/io;
 public function main() {
     io:println("Iterating over a string array:-");
     string[] fruits = ["apple", "banana", "cherry"];
-    // To fetch the values defined in the array, use one variable.
-    // To fetch the index and value, use two variables as comma separated values. E.g., `foreach i, v in fruits`.
+
+    // Foreach statement can be used to iterate over an array. Each iteration returns an element in the array. Note
+    // that the index of the corresponding element is not returned.
     foreach var v in fruits {
         io:println("fruit: " + v);
     }
 
     io:println("\nIterating over a map:-");
     map<string> words = { a: "apple", b: "banana", c: "cherry" };
-    // To fetch the values defined in the map, use one variable. To fetch
-    // both the key (`string`) and value, use two variables.
+
+    // Each iteration will return the key (`string`) and the value as a tuple variable. Use tuple destructuring to
```

Comment:
```suggestion
    // Each iteration will return the key (`string`) and the value as a `tuple` variable. Use tuple destructuring to
```

Shall we add "tuple" between single back ticks as well? Apply everywhere if applicable.